### ISSUE TITLE
Differenciate Anonymous Users by Random Number

### ISSUE THREAD

**Issue by Satoshi-Sh** [nickytonline/astro-partykit-starter#42]
## Is your feature request related to a problem? Please describe.

After [this PR](https://github.com/nickytonline/astro-partykit-starter/pull/38), users can join a chat room as anonymous users without putting usernames. However, if more than one "anonymous" in a chat room, it's hard to follow the conversation. 

![Screenshot from 2024-01-14 13-05-13](https://github.com/nickytonline/astro-partykit-starter/assets/73622805/38c6c415-bda3-410e-955b-0d2327f58928)

## Describe the solution you'd like

I would like to add some random number to "anonymous" like "anonymous_1234"

#### src/pages/chat/index.astro
````
const randomNum = Math.floor(Math.random() *10000).toString()
const username = !usernameParams ? `anonymous_${randomNum}` : usernameParams; 
````
Uuid can be used for this, but it would be too long for a username. I think a random number(0-9999) would be good enough for this application.
```
let uuid = self.crypto.randomUUID();
console.log(uuid); // for example "36b8f84d-df4e-4d49-b662-bcde71a8764f"
```

**Comment by nickytonline** [comment]
Another option could be to use UUID, but that means installing another package.

```
import { v4: uuidv4 } from 'uuid';

const randomUUID = uuidv4();
```
